packages/transport-socket/package.json

```diff
@@ -68,6 +68,7 @@
   "dependencies": {
     "@stellarjs/core": "^0.9.1",
     "bluebird": "^3.4.7",
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "uuid": "^3.1.0"
```

Comment:
please sync versions with backoffice to avoid duplications (if needed)

Reply:
they are the same